sdk/python/feast/feature_store.py

```diff
@@ -1915,7 +1916,7 @@ def retrieve_online_documents(
             online_features_response=online_features_response,
             data={
                 **join_key_values,
-                requested_feature: document_feature_vals,
+                "embedding": document_feature_vals,  # Replace the hardcoded "embedding" with the actual feature name
```

Comment:
yeah this was a hack at first for some reason with sqlite early on i believe.

Reply:
It got patched in a later version.

Reply:
I don't see it's patched https://github.com/feast-dev/feast/blob/master/sdk%2Fpython%2Ffeast%2Ffeature_store.py#L1918